src/main/java/org/sagebionetworks/web/client/widget/header/HeaderViewImpl.java

```diff
@@ -83,6 +85,10 @@ public void clear() {
 
 
 	public void initClickHandlers() {
+		editModeNavBarClickBlocker.addClickHandler(event -> {
```

Comment:
This is a layer on top of the nav bar where we can catch all click events (and set the background color to grey it out)